src/assistant-markdown.css

```diff
@@ -0,0 +1,96 @@
+@import 'highlight.js/styles/github.css';
+
+:root {
+	--gh-bg-color: #f6f8fa;
+	--gh-text-color: #24292e;
+	--gh-border-color: #e1e4e8;
+	--gh-blockquote-border-color: #d1d5da;
+}
+
+#assistant-markdown {
+	line-height: 1.6;
+	word-wrap: break-word;
+}
+
+#assistant-markdown code {
+	border-radius: 3px;
+	white-space: pre-wrap;
+	margin: 1rem 0 !important;
+	background-color: var(--gh-bg-color);
+	color: var(--gh-text-color);
+}
+
+#assistant-markdown pre {
+	padding: 1.5rem;
+	border-radius: 5px;
+	overflow-x: auto;
+	margin: 1rem 0;
+	background-color: var(--gh-bg-color);
+	color: var(--gh-text-color);
+}
+
+#assistant-markdown hr {
+	border: none;
+	margin: 1rem 0;
+	border-top: 1px solid var(--gh-border-color);
+}
+
+#assistant-markdown ul,
+#assistant-markdown ol {
+	margin: 1rem 0;
+	padding-left: 1.5rem;
+}
+
+#assistant-markdown ul {
+	list-style-type: disc;
+}
+
+#assistant-markdown ol {
+	list-style-type: decimal;
+}
+
+#assistant-markdown blockquote {
+	margin: 1rem 0;
+	padding: 0.5rem 1rem;
+	background-color: var(--gh-bg-color);
+	border-left: 0.25rem solid var(--gh-blockquote-border-color);
+	color: var(--gh-text-color);
+	font-weight: bold;
+}
+
+#assistant-markdown h1,
+#assistant-markdown h2,
+#assistant-markdown h3,
+#assistant-markdown h4,
+#assistant-markdown h5,
+#assistant-markdown h6 {
+	margin-top: 1.5rem;
+	margin-bottom: 0.5rem;
+	color: var(--gh-text-color);
+	font-weight: bold;
+}
+
+#assistant-markdown h1 {
+	font-size: 1.5rem;
+}
+
+#assistant-markdown h2 {
+	font-size: 1.25rem;
+}
+
+#assistant-markdown h3 {
+	font-size: 1.125rem;
+}
+
+#assistant-markdown h4 {
+	font-size: 1rem;
+	margin-bottom: 0.5rem;
+}
+
+#assistant-markdown h5 {
+	font-size: 0.875rem;
+}
+
+#assistant-markdown h6 {
+	font-size: 0.85rem;
+}
```

Comment:
The `id` scope to `#assistant-markdown` is intentional -- I feel an `id` CSS scope is appropriate to avoid any other global styling conflicts.

Also open to feedback on code organization -- eventually, I think there may be a better place for this file to live than in the root of the `src` directory.